control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -133,6 +135,20 @@ async fn issue_fs_unfreeze(endpoint: String, volume_id: String) -> Result<(), St
     }
 }
 
+#[tracing::instrument]
+async fn issue_cleanup(endpoint: String, volume_id: String) -> Result<(), Status> {
+    tracing::info!("Issuing cleanup of stale entries before publish for volume: {volume_id}");
```

Comment:
How about removing caller side tracing and only have this one to print node and endpoint also, by taking app node as parameter in this function.

Reply:
Done.